Formula/fontconfig.rb

```diff
@@ -29,6 +29,7 @@ class Fontconfig < Formula
   end
 
   depends_on "pkg-config" => :build
+  depends_on "python@3.10" => :build if MacOS.version <= :high_sierra
```

Comment:
Isn't the better way to do this
```ruby
uses_from_macos "python" => :build, since: :mojave
```
?

Reply:
People are reporting this issue on Mojave despite it having `/usr/bin/python3` so I think it's going to be easier to use our Python everywhere (and is consistent to what we do elsewhere anyway).

Reply:
> People are reporting this issue on Mojave despite it having `/usr/bin/python3` so we'll need to either just use our Python (like we do elsewhere) or investigate that further.

I suddenly realized I'm using CommandLineTools 10.14 despite High Sierra (10.13). Looking at [Homebrew/brew/#13111](https://github.com/Homebrew/brew/issues/13111) that guy's using CLT 10.14 too, while `brew doctor` suggests he install CLT **10.15** which was for Catalina. In CLT10.14 there simply isn't a `python3` to begin with!

So it should be either uses_from_macos if CLT version >= 10.15 or just python@3.10 unconditionally for Mojave and below. Which one do you prefer?